sources/model/Cloud.cpp

```diff
@@ -117,6 +118,10 @@ Cloud::Cloud(VecSceneSound *soundSet, float theNumGrains)
     // set volume of cloud to unity
     //setVolumeDb(0.0);
     setVolumeDb(g_defaultCloudParams.volumeDB);
+    envelopeVolume = new Env;
+    envelopeVolume->setParam(g_defaultCloudParams.envelope);
+    envelopeVolumeBuff = new float[g_buffSize];
+    intermediateBuff = new double[g_buffSize * MY_CHANNELS];
```

Comment:
Ne pas oublier de libérer celui-ci également.

Reply:
oui, ça j'avais noté de devoir le faire, en effet